app/assets/stylesheets/components/page_specific/_promo.scss

```diff
@@ -9,19 +9,73 @@
 .promo__heading {
   @extend %font-heading-medium;
   @include body(18, 24);
-  margin-top: 0;
+  margin: 0 0 $baseline-unit;
+  position: relative;
 
   @include respond-to($mq-m) {
     @include body(20, 24);
   }
+
+  color: $color-blue-medium;
+
+  .l-tool-promos__item & {
+    padding-right: $baseline-unit*3;
+  }
+}
+
+.promo__heading--blog:after {
+  background: $color-mas-blog-blue;
+  border-radius: 4px;
+  bottom: 0;
+  color: #fff;
+  display: inline-block;
+  font-size: 18px;
+  font-weight: 700;
+  height: 30px;
+  content: "Blog";
```

Comment:
@benbarnett are you referring to the content: "Blog" ? - this is to add the blue tag 'blog' to the images that need it.